state.go

```diff
@@ -356,24 +356,52 @@ func (i *InitState) Handle(ctx *StateContext, c Controller) (State, bool) {
 
 	log.Infof("handling loaded state: %s", sd.Name)
 
-	// chack last known state
-	switch sd.Name {
-	// update process was finished; check what is the status of update
-	case MenderStateReboot:
+	if !committedPartition(c) {
+		// only valid entrypoint into the uncommitted partition is reboot_leave
```

Comment:
Is `commit_enter` also a possibility here? What I am thinking is we've entered `commit_enter` state and then the reboot happened. Will we end up here as well?